aries_cloudagent/holder/routes.py

```diff
@@ -292,6 +455,13 @@ async def register(app: web.Application):
             ),
             web.delete("/credential/{credential_id}", credentials_remove),
             web.get("/credentials", credentials_list, allow_head=False),
+            web.get(
+                "/credential/dif/{credential_id}",
+                dif_cred_get,
+                allow_head=False,
+            ),
+            web.delete("/credential/dif/{credential_id}", dif_cred_remove),
+            web.get("/credentials/dif", dif_creds_list, allow_head=False),
```

Comment:
`TypeError: Failed to execute 'fetch' on 'Window': Request with GET/HEAD method cannot have body.`